postgres/up/05-create_filesystem_table.sql

```diff
@@ -13,18 +13,42 @@ CREATE TABLE filesystem (
   IsPublished   BOOLEAN DEFAULT false,
   CreatedAt     TIMESTAMP NOT NULL DEFAULT NOW(),
 
+  /* MetaData */
+  MetaID        uuid NOT NULL,
+
   OwnedBy       INT,
   /* Pain */
   Parent        uuid REFERENCES filesystem(EntityID) DEFAULT NULL,
 
   /* FK Constraint */
   CONSTRAINT fk_owner FOREIGN KEY (OwnedBy) 
     REFERENCES groups(UID),
+
   /* Unique name constraint: there should not exist an entity of the same type with the
      same parent and logical name. */
   CONSTRAINT unique_name UNIQUE (Parent, LogicalName, IsDocument)        
 );
 
+/**
+  Metadata table
+**/
+DROP TABLE IF EXISTS metadata;
+CREATE TABLE metadata (
+  
+  MetaID        uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+  EntityID      uuid NOT NULL,
+  CreatedAt     TIMESTAMP NOT NULL DEFAULT NOW(),
+
+  CONSTRAINT fk_file FOREIGN KEY (EntityID)
+    REFERENCES filesystem(EntityID),
```

Comment:
We do we have a foreign key from Filesystem -> Meta and Meta -> Filesystem? Realistically we will only ever be accessing metadata when querying `Filesystem` and consequentially would perform a join on `MetaID`, I dont think the reference from Meta -> Filesystem is required